### PR TITLE
Add ESLint linter from tagview repository

### PR DESCRIPTION
**Pull request by jiparis**
I've merged latest changes from tagview/arcanist-extensions so that we get the latest eslint linter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,4 +176,44 @@ Below is an example of an `.arclint` file that includes the SCSS-Lint Linter:
...
+This extension will lint your project using [ESlint](http://eslint.org). It is important to mention that the extension won't install ESlint, so you must do it manually. Just make sure you have the `eslint` executable listed on your `$PATH`.
+
+Below is an example of an `.arclint` file that includes the ESlint Linter:
+
+```json
+{
+  "linters": {
+    "javascript": {
+      "type": "eslint",
+      "include": "/\\.js$/"
```
Comment (Angelmmiguel) [inline README.md]: I think we should include `.jsx` files too:

```
"include": "/\\.(js|jsx)$/"
```

Comment (jiparis) [inline README.md]: Yes, see .arclint changes. This is just the README file (merged from main repository)

Comment (Angelmmiguel) [inline README.md]: 👍

**Comment by prydonius** [comment]
lgtm :+1:

**Pull request merged**